README.md

```diff
@@ -1,17 +1,17 @@
 # :octocat: GitHub templates
 A collection of GitHub issue and pull request (PR) templates
 
-## Multiple Issue and PR Templates
+## Multiple issue and PR Templates
 GitHub supports [multiple issue templates](https://help.github.com/articles/about-issue-and-pull-request-templates/).
 
-## What is an issue and PR template?
-[Issue and Pull Request templates](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/)
+## What are issue and PR templates?
+[Issue and pull request templates](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/)
 
 ## How to create issue and PR templates?
 [About issue and pull request templates](https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates)
 
-## License and Copyright
-Copyright (c) 2020 Inpsyde GmbH.
+## License and copyright
+Copyright (c) 2021 Inpsyde GmbH.
```

Comment:
we should never add the current year here. Because then we need to update every year the year to match the current year 😬